gratipay/models/exchange_route.py

```diff
@@ -53,12 +49,14 @@ def from_address(cls, participant, network, address):
         return r
 
     @classmethod
-    def insert(cls, participant, network, address, error='', fee_cap=None, cursor=None):
+    def insert(cls, participant, network, address, error='', fee_cap=None, cursor=None,
+                                                                                 is_deleted=False):
```

Comment:
Hmm, I don't think it is worth adding `is_deleted` to the `insert` function. No one (other than in tests) is going to insert a deleted route, are they